scripts/run_cass_and_test.sh

```diff
@@ -11,8 +11,21 @@ trap finish EXIT
 # shut down containers
 docker-compose -f docker/docker-compose.yml down;
 
-# run cassandra container
-docker-compose -f docker/docker-compose.yml up -d cassandra;
+# run cassandra & cadence container
+# we need cadence here because it handles db setup
+docker-compose -f docker/docker-compose.yml up -d cassandra cadence;
+
+status=""
+while [[ "$status" != "running" ]]; do
+  echo "waiting for containers to be healthy. status: $status"
+  sleep 5
+  # checking cadence container is running is sufficient because it depends on health status of cassandra in docker-compose.yml
+  status="$(docker inspect docker-cadence-1 -f '{{ .State.Status }}')"
+done;
+
+echo "containers are healthy. sleeping for 10 seconds so cadence container can finish db setup"
+sleep 10
```

Comment:
I guess it will work until it won't.
This should be ok to merge, but I think we need to implement healthcheck on docker container so you can just wait for container statys "healthy" instead of sleep

Reply:
yes that would be ideal. this is a script for development purposes and not part of CI. so flakyness expectations is low :)